src/invert.ts

```diff
@@ -0,0 +1,35 @@
+import { purry } from './purry';
+
+type Invertable = { [index: string]: string } | { [index: number]: string };
+type Inverted = { [index: string]: string };
```

Comment:
as @texastoland has pointed out, I think the types could be improved here to have a relation with each other. The generic `T` you've added to `invert` is actually unused ...